packages/fx-core/src/core/middleware/envInfoLoader.ts

```diff
@@ -43,7 +43,7 @@ import { newEnvInfo } from "../tools";
 import { CoreHookContextV2 } from "../v2";
 
 const newTargetEnvNameOption = "+ new environment";
-const lastUsedMark = " (activate)";
+const lastUsedMark = " (active)";
```

Comment:
nit: the variable name can be updated for its new meaning, maybe `activeMark` or something better.

Reply:
Sounds good. Updated.